### DISCUSSION TITLE
<TabItems> Renders Anything But Text as a Code Block?

### DISCUSSION THREAD

**Discussion by nategoethel** [facebook/docusaurus#5500] [Q&A] [answered]
Hi,

Do `<TabItem>` elements support only text? When I try to add something like an ordered list (in Markdown), it's all rendered as a code block:
![Screen Shot 2021-09-07 at 3 20 52 PM](https://user-images.githubusercontent.com/44580393/132405845-f11388d0-09fc-4415-b7ce-ad6983c57fed.png)

Here is the MDX:
```html
<Tabs groupId="views">
  <TabItem value="chat" label="Chat" default>
    You can load data from a file you've uploaded by entering Load data from the file file name.
  </TabItem>
  <TabItem value="sheets" label="Sheets">
    To load data into your session:

    1. Example Step 1.

    2. Example Step 2

    3. Example Step 3
    
    1. a
    2. b
    3. c

  </TabItem>
  <TabItem value="search" label="Search">
    This is a banana
  </TabItem>
</Tabs>
```
Is there a way to have the Markdown render as it does outside of the `<Tabs>` and `<TabItems>`?

**Comment by Josh-Cena** [comment, marked as answer]
You shouldn't have indented your text. Per Markdown spec, any indented text blocks become code blocks, and JSX-interleaved syntax is not exempted from this.

```jsx
<Tabs groupId="views">
  <TabItem value="chat" label="Chat" default>
    You can load data from a file you've uploaded by entering Load data from the file file name.
  </TabItem>
  <TabItem value="sheets" label="Sheets">

To load data into your session:

1. Example Step 1.

2. Example Step 2

3. Example Step 3
    
1. a
2. b
3. c

  </TabItem>
  <TabItem value="search" label="Search">
    This is a banana
  </TabItem>
</Tabs>
```

It looks ugly, I know, but it's the only correct way. Maybe we should document this better?

**Reply by nategoethel** [reply]
Thanks, but this fails to compile:
![Screen Shot 2021-09-07 at 7 02 41 PM](https://user-images.githubusercontent.com/44580393/132425013-02760341-20cf-49b7-95e2-1a880ed725d2.png)

**Reply by Josh-Cena** [reply]
Wow, wasn't expecting that, but it turns out you need one extra line break:

```jsx
<Tabs groupId="views">
  <TabItem value="chat" label="Chat" default>
    You can load data from a file you've uploaded by entering Load data from the file file name.
  </TabItem>
  <TabItem value="sheets" label="Sheets">

To load data into your session:

1. Example Step 1.

2. Example Step 2

3. Example Step 3
    
1. a
2. b
3. c


  </TabItem>
  <TabItem value="search" label="Search">
    This is a banana
  </TabItem>
</Tabs>
```
